### PR TITLE
[chore] readme updated and removed ghcr reference for image.

### PR DESCRIPTION
**Pull request by adityagajbhiye9**
## Description

-  readme updated and removed ghcr reference for image.

#72 - fixed.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by sachinargade123 — APPROVED** [review]
LGTM

**Review by hzierer — APPROVED** [review]
LGTM

File: .github/workflows/dockerbuild.yml [hunk 1]
```diff
@@ -99,3 +99,5 @@ jobs:
           username: ${{ secrets.DOCKER_HUB_USER }}
           password: ${{ secrets.DOCKER_HUB_TOKEN }}
           repository: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
+          # IMPORTANT: Adjust this to the actual path of your container image notice
+          readme-filepath: ./docker_notice.md
```
Comment (FaGru3n) [inline .github/workflows/dockerbuild.yml:103]:
```suggestion
          readme-filepath: ./DOCKER_NOTICE.md
```

**Review by FaGru3n — CHANGES_REQUESTED** [review]
same with the other Repo for Upper Case DOCKER_NOTICE

**Review by FaGru3n — APPROVED** [review]

**Pull request merged**